src/getAppSyncConfig.js

```diff
@@ -186,9 +187,17 @@ export default function getAppSyncConfig(context, appSyncConfig) {
     }, []);
   };
 
+  // Load the schema. If multiple provided, merge them
+  const schemaPaths = Array.isArray(cfg.schema) ? cfg.schema : [cfg.schema || 'schema.graphql'];
+  const schemas = schemaPaths.map((schemaPath) => getFileMap(context.serverless.config.servicePath, schemaPath));
+  const schema = {
+    path: schemas.find((schema) => schema.path),
+    content: mergeTypes(schemas.map((schema) => schema.content)),
+  };
+
   return {
     appSync: makeAppSync(cfg),
-    schema: getFileMap(context.serverless.config.servicePath, cfg.schema || 'schema.graphql'),
+    schema: schema, 
```

Comment:
@EgorDm can you use the short-hand `schema,` here? (Did you run eslint on your code? )

Reply:
@bboure Indeed a good one. I have run the code through the eslint and fixed some issues there. This should be covered in the new commit.